pages/lua_manual/api_reference.md

```diff
@@ -0,0 +1,1849 @@
+---
+layout: page
+title: Lua API Reference
+status: improve
+---
+
+All instantiable classes define a `new()` method which returns an object of that particular class. 
+
+With the exception of this `new()` method, all members listed will be member methods.
+
+## ElementDataGrid
+
+Inherits: `Element`{: .type }
+
+ElementDataGrid derives from Element. The data grid has the following methods and properties:
+
+### Properties
+
+| Types | Name |
+| ------------ | ---- |
+| `ElementDataGridRow`{: .type } | rows |
+
+
+### Methods
+
+| Return value | Name |
```

Comment:
I think you mean something like this:

|Member|Type|
|------|----|
|some_property|Property|
|int SomeMethod(int param1, int param2)|Method|
|__add|Metafunction|

This could get a lot crowded when there are many function arguments and long return types. And to fix this we could just mention the function name and nothing else, but that is kind of going against the motive here I think.

Reply:
Alright, I'll trust you with this, looks pretty good now.